modules/nw-egress-ips-about.adoc

```diff
@@ -65,7 +65,9 @@ When creating an `EgressIP` object, the following conditions apply to nodes that
 
 - An egress IP address is never assigned to more than one node at a time.
 - An egress IP address is equally balanced between available nodes that can host the egress IP address.
-- If the `spec.EgressIPs` array in an `EgressIP` object specifies more than one IP address, no node will ever host more than one of the specified addresses.
+- If the `spec.EgressIPs` array in an `EgressIP` object specifies more than one IP address then the following conditions apply:
```

Comment:
```suggestion
- If the `spec.EgressIPs` array in an `EgressIP` object specifies more than one IP address, the following conditions apply:
```
Suggestion: Avoid using "then" to introduce an independent clause that follows an "if" clause.